Lib/posixpath.py

```diff
@@ -415,8 +390,8 @@ def realpath(filename, *, strict=False):
     path, ok = _joinrealpath(filename[:0], filename, strict, {})
     return abspath(path)
 
-# Join two paths, normalizing and eliminating any symbolic links
-# encountered in the second path.
+# Join two paths, normalizing and eliminating any symbolic links encountered in
+# the second path. Two leading slashes are replaced by a single slash.
```

Comment:
To keep a minimal diff here, I'd prefer it if you did this, so that it's a single line being added to the comment rather than two existing lines being edited:

```suggestion
# Join two paths, normalizing and eliminating any symbolic links
# encountered in the second path.
# Two leading slashes are replaced by a single slash.
```